src/pages/BuyGMX/BuyGMX.tsx

```diff
@@ -226,10 +234,11 @@ function CentralisedExchanges({ chainId, externalLinks }) {
             <Trans>Buy GMX from centralized exchanges:</Trans>
           </div>
           <div className="buttons-group">
-            {CENTRALISED_EXCHANGES.filter((e) => e.networks.includes(chainId)).map((exchange) => {
+            {CENTRALISED_EXCHANGES.filter((e) => Object.keys(e.links).includes(String(chainId))).map((exchange) => {
```

Comment:
Hm, looks a bit strange casting chainId to String here
will just `chainId in e.links` work?

Reply:
Done!